### PR TITLE
Feature/attributed string

### PR DESCRIPTION
**Pull request by zmeriksen**
#66 #65 #64

### CODE REVIEW

**Comment by zmeriksen** [comment]
Example Updated: https://github.com/zmeriksen/SwiftUIKit_Test/pull/4/files

File: Sources/SwiftUIKit/Views/Label.swift [hunk 1]
```diff
@@ -51,6 +62,23 @@ public class Label: UILabel {
         
         return self
     }
+    
+    @discardableResult
+    public func apply(attributes: StringAttributes) -> Self {
+
```
Comment (ambid17) [inline Sources/SwiftUIKit/Views/Label.swift]: inconsistent spacing

File: Sources/SwiftUIKit/Views/Label.swift [hunk 2]
```diff
@@ -22,6 +22,17 @@ public class Label: UILabel {
         accessibility(label: text, traits: .staticText)
     }
     
+    public init(_ attributedText: AttributedString) {
+        super.init(frame: .zero)
+        
+        self.attributedText = attributedText
+        
+        if #available(iOS 10.0, *) {
+            adjustsFontForContentSizeCategory = true
```
Comment (ambid17) [inline Sources/SwiftUIKit/Views/Label.swift]: should this be toggle-able?

Comment (zmeriksen) [reply Sources/SwiftUIKit/Views/Label.swift]: You can configure this until there is a reported issue.

**Pull request merged**